Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller.py

```diff
@@ -6,11 +6,26 @@
 SCRIPT_NAME = 'CustomPackInstaller'
 
 
-def install_custom_pack(pack_id: str) -> Tuple[bool, str]:
+def build_url_parameters(skip_verify: bool, skip_validation: bool) -> str:
+    is_server_ge_to_6_5 = is_demisto_version_ge('6.5.0')
+    is_server_ge_to_6_6 = is_demisto_version_ge('6.6.0')
+
+    uri = '/contentpacks/installed/upload'
+    if skip_verify == 'true' and is_server_ge_to_6_5:
+        uri = f'{uri}?skipVerify=true'
+
+    if skip_validation == 'true' and is_server_ge_to_6_6:
+        uri = f'{uri}&skipValidation=true' if '?' in uri else f'{uri}?skipValidation=true'
```

Comment:
please refactor as we discussed to make this more readable

Reply:
done